code/datums/jobs.dm

```diff
@@ -904,6 +910,8 @@ ABSTRACT_TYPE(/datum/job/civilian)
 	slot_ears = list(/obj/item/device/radio/headset/civilian)
 	items_in_backpack = list(/obj/item/device/camera_viewer/ranch,/obj/item/storage/box/knitting)
 
+	faction = FACTION_BOTANY
```

Comment:
Is rancher botany? Currently wasps and such only care about specifically botanists
![image](https://github.com/goonstation/goonstation/assets/20713227/6a5fa9d7-b4b9-42ed-88fe-442e37beeda0)

Reply:
Ok removed the faction from rancher and added to the beekeeper instead since they get wasp grenades